jobs/api/v4/JobsSample/CustomRankingSearchSample.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using Google.Cloud.Talent.V4Beta1;
+using static Google.Cloud.Talent.V4Beta1.SearchJobsRequest.Types;
+using static Google.Cloud.Talent.V4Beta1.SearchJobsRequest.Types.CustomRankingInfo.Types;
+
+namespace GoogleCloudSamples
+{
+    internal class CustomRankingSearchSample
+    {
+        // [START job_search_custom_ranking_search]
+        public static object CustomRankingSearch(string projectId, string tenantId)
+        {
+            JobServiceClient jobServiceClient = JobServiceClient.Create();
+            TenantName name = new TenantName(projectId, tenantId);
+
+            string domain = "www.example.com";
+            string sessionId = "Hashed session identifier";
+            string userId = "Hashed user identifier";
+            RequestMetadata requestMetadata = new RequestMetadata
+            {
+                Domain = domain,
+                SessionId = sessionId,
+                UserId = userId
+            };
+
+            CustomRankingInfo customRankingInfo = new CustomRankingInfo
+            {
+                ImportanceLevel = ImportanceLevel.Extreme,
+                RankingExpression = "(someFieldLong + 25) * 0.25"
```

Comment:
It's not clear to me what is someFieldLong. Consider adding a comment or a more descriptive name.